db/upgrade_migrations/20160723012349_upgrade_v062_to_v063.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class UpgradeV062ToV063 < ActiveRecord::Migration[5.0]
+  def up
+    i = 1
+    Thredded::MessageboardGroup.order(created_at: :desc).each do |group|
+      Rails.logger.info group.id
+      next if group.valid?
+
+      group.name = "#{group.name}-#{i}"
+      group.save!
+      i += 1
+    end
```

Comment:
Generally the migration should only be run with the version of the gem it is migrating to. However, many will run it on the version of the gem it's migrating from, so it'd be nice if it worked there as well.

Using only plain activerecord features I think is fine, because these depend only on the schema. However, using `valid?` is dangerous because we might change the validation in a minor/patch version and then this migration will no longer do the expected thing if the record is invalid due to some other reason.